python/src/aac/lang/definitions/extensions.py

```diff
@@ -106,18 +106,34 @@ def _remove_schema_fields(target_fields: list, fields_to_remove: list):
 def _add_extension_required_fields_to_defintion(target_definition_fields: dict, extension_additional_content_fields: dict) -> None:
     """Add any additional required fields from the extension to the target definition."""
     extension_required_fields = extension_additional_content_fields.get("required")
+    definition_validations = target_definition_fields.get("validation") or []
+
+    if not definition_validations:
+        target_definition_fields["validation"] = []
+
+    from aac.plugins.validators.required_fields import REQUIRED_FIELDS_VALIDATION_STRING
```

Comment:
I hate doing this but I haven't found another way to resolve the circular import issue. I think we're going to need to put some extra thought into how we handle required fields beyond what we originally planned to do.

Reply:
I think this is an ok solution, it's not really any worse than the earlier idea. However, I'd make it a private function rather than writing it inline twice.